airflow/www_rbac/decorators.py

```diff
@@ -32,27 +32,7 @@ def action_logging(f):
     """
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
-
-        with create_session() as session:
-            if g.user.is_anonymous:
```

Comment:
What is g? I'm curious if we can use this on the api requests as well rather than you building the logic there directly

Reply:
My assumption is that it's session based so it won't work?

Reply:
yeah i tried this initially and it didn't work. traced it back and realized that the API auth isn't tied into flask, its just a decorator that queries the metadata tables